src/ec/packages/ec-component-social-media-share/demo/data.js

```diff
@@ -0,0 +1,96 @@
+const demoData = {
```

Comment:
it is better to base the demo data on the data provided by the specs (when possible).
also some parameters here are the same every time (`size: 'xl'` and `extra-classes` for icons for instance). So they could be put directly in the component's twig.